tuf-spec.md

```diff
@@ -996,6 +996,10 @@ repo](https://github.com/theupdateframework/specification/issues).
 
 ### **The client application**
 
+  Note: If a step in the following workflow does not succeed (e.g., a failure
+  is reported because a new metadata file was not signed), the client should
+  not be left in an unrecoverable state.
+
```

Comment:
Two points:

1. `client should not be left in an unrecoverable state` seems less specific than it could be. Consider instead: `the client must be able to attempt update again`?

2. Steps below say "report the signature failure" or "report ... attack", but reporting doesn't necessarily mean aborting the update cycle. Would we want implementers to continue with existing unexpired versions of metadata? If timestamp fails to validate, or a new version of root fails to validate, should we continue the update cycle using the trusted unexpired versions? I think no. So consider "If any metadata fails to validate, abort the update cycle".